.nvmrc

```diff
@@ -0,0 +1 @@
+20
```

Comment:
Is this `.nvmrc` necessary?

Reply:
As I understand it, the [action.yml defines the node version used when it executes](https://github.com/anchore/sbom-action/blob/main/action.yml#L81), does it not?

Reply:
The `action.yml` uses its own config to set NodeJS version.

The `.nvmrc` is for tooling, eg. VS Code.